crates/dbs-virtio-devices/src/fs/device.rs

```diff
@@ -480,15 +474,14 @@ impl<AS: GuestAddressSpace> VirtioFs<AS> {
         prefetch_list_path: Option<String>,
     ) -> FsResult<()> {
         debug!("http_server rafs");
-        let mut file = <dyn RafsIoRead>::from_file(&source)
-            .map_err(|e| FsError::BackendFs(format!("RafsIoRead failed: {:?}", e)))?;
+        let mut file = Path::new(&source);
         let (mut rafs, rafs_cfg) = match config.as_ref() {
             Some(cfg) => {
-                let rafs_conf: RafsConfig =
-                    serde_json::from_str(cfg).map_err(|e| FsError::BackendFs(e.to_string()))?;
+                let rafs_conf: Arc<ConfigV2> =
+                    Arc::new(serde_json::from_str(cfg).map_err(|e| FsError::BackendFs(e.to_string()))?);
 
                 (
-                    Rafs::new(rafs_conf, mountpoint, &mut file)
+                    Rafs::new(&rafs_conf, mountpoint, &mut file)
```

Comment:
Possible without the reference?

Reply:
Nope, `Rafs` api has changed in the new Nydus.